### PR TITLE
[Refactor] Refactor Starrocks LOG to reduce the log file size(part3)

### PR DESCRIPTION
**Pull request by sevev**
## Why I'm doing:
Refactor Starrocks LOG to reduce the log file size.

## What I'm doing:
Refactor Starrocks FE LOG to reduce the log file size.

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [x] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.3
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java [hunk 1]
```diff
@@ -467,7 +467,7 @@ public VisibleStateWaiter commitPreparedTransaction(long transactionId) throws U
             } finally {
                 updateCatalogAfterCommittedSpan.end();
             }
-            LOG.info("transaction:[{}] successfully committed", transactionState);
+            LOG.debug("transaction:[{}] successfully committed", transactionState);
```
Comment (gengjun-git) [inline fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java:470]: I think this log needs to keep.

File: fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java [hunk 2]
```diff
@@ -1900,7 +1900,7 @@ public void finishTransactionNew(TransactionState transactionState, Set<Long> pu
         // do after transaction finish
         GlobalStateMgr.getCurrentState().getOperationListenerBus().onStreamJobTransactionFinish(transactionState);
         GlobalStateMgr.getCurrentState().getLocalMetastore().handleMVRepair(transactionState);
-        LOG.info("finish transaction {} successfully", transactionState);
+        LOG.debug("finish transaction {} successfully", transactionState);
```
Comment (gengjun-git) [inline fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java:1903]: I think this log needs to keep.

File: fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java [hunk 3]
```diff
@@ -1769,22 +1769,22 @@ public void replayUpsertTransactionState(TransactionState transactionState) {
...
-                LOG.info("remove unknown transaction: {}", transactionState);
+                LOG.debug("remove unknown transaction: {}", transactionState);
                 return;
             }
             // set transaction status will call txn state change listener
             transactionState.replaySetTransactionStatus();
             Database db = globalStateMgr.getLocalMetastore().getDb(transactionState.getDbId());
             if (transactionState.getTransactionStatus() == TransactionStatus.COMMITTED) {
-                LOG.info("replay a committed transaction {}", transactionState);
+                LOG.debug("replay a committed transaction {}", transactionState);
```
Comment (gengjun-git) [inline fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java:1779]: I think this log needs to keep.

File: fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java [hunk 4]
```diff
@@ -1769,22 +1769,22 @@ public void replayUpsertTransactionState(TransactionState transactionState) {
...
                 updateCatalogAfterCommitted(transactionState, db);
             } else if (transactionState.getTransactionStatus() == TransactionStatus.VISIBLE) {
-                LOG.info("replay a visible transaction {}", transactionState);
+                LOG.debug("replay a visible transaction {}", transactionState);
```
Comment (gengjun-git) [inline fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java:1782]: I think this log needs to keep.

File: fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java (hunk 1 above)
Comment (sevev) [reply fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java]: > I think this log needs to keep.

done.

File: fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java (hunk 3 above)
Comment (sevev) [reply fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java:1783]: > I think this log needs to keep.

done.

File: fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java (hunk 2 above)
Comment (sevev) [reply fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java]: > I think this log needs to keep.

only ignore this log in checkpoint thread right now, PTAL again.

File: fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java (hunk 4 above)
Comment (sevev) [reply fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java:1789]: > I think this log needs to keep.

same as the above.

**Review by gengjun-git — APPROVED** [review]

**Review by ZiheLiu — APPROVED** [review]

**Comment by sevev** [comment]
@mergify backport branch-3.2

**Comment by sevev** [comment]
@mergify backport branch-3.3

**Pull request merged**